File: baekjoon/2309.py
a = [int(input()) for _ in range(9)]
aim=sum(a)-100
a.sort()
def solve():
    for i in range(9):
        for j in range(i+1,9):
            if aim==a[i]+a[j]:
                for k in range(9):
                    if k==i or k==j:
                        continue
                    else:
                        print(a[k])
                return
solve()

File: baekjoon/2309.cpp
#include <algorithm>
#include <iostream>
#pragma warning(disable : 4996)
using namespace std;
int main()
{
    int a[9];
    int sum = 0;
    for (int i = 0; i < 9; i++)
    {
        cin >> a[i];
        sum += a[i];
    }
    sort(a, a + 9);
    sum = sum - 100;
    int i = 0;
    int j = 0;
    for (i = 0; i < 8; i++)
    {
        for (j = i + 1; j < 9; j++)
        {
            if (a[i] + a[j] == sum)
            {
                for (int k = 0; k < 9; k++)
                {
                    if (k == i || k == j)
                        continue;
                    cout << a[k] << endl;
                }
                return 0;
            }
        }
    }
}